sdk/eventgrid/Azure.Messaging.EventGrid/src/MigrationGuide.md

```diff
@@ -0,0 +1,195 @@
+# Guide for migrating to Azure.Messaging.EventGrid from Microsoft.Azure.EventGrid
+
+This guide is intended to assist in the migration to Azure.Messaging.EventGrid from Microsoft.Azure.EventGrid. It will focus on side-by-side comparisons for similar operations between the two packages.
+
+We assume that you are familiar with Microsoft.Azure.EventGrid. If not, please refer to the README for [Azure.Messaging.EventGrid](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventgrid/Azure.Messaging.EventGrid/README.md) rather than this guide.
```

Comment:
```suggestion
We assume that you are familiar with `Microsoft.Azure.EventGrid`. If not, please refer to the [README](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventgrid/Azure.Messaging.EventGrid/README.md) for `Azure.Messaging.EventGrid` rather than this guide.
```